mods/_lott/lottfarming/crafting.lua

```diff
@@ -30,7 +30,8 @@ local function decaying_wood(pos)
 	local node = minetest.get_node(pos)
 	local name = node.name
 	local above = minetest.get_node({x=pos.x, y=pos.y+1, z=pos.z})
-	if name == "default:tree_trunk" or name == "default:jungletree_trunk" then
+	if name == "default:tree" or name == "default:jungletree" or
+	name == "default:tree_trunk" or name == "default:jungletree_trunk" then
```

Comment:
Какое-то уже большое условие. Может вынести его в переменную, чтобы `if` лучше был виден?
```suggestion
	local can_decay = name == "default:tree" or name == "default:jungletree" or
		name == "default:tree_trunk" or name == "default:jungletree_trunk"
	if can_decay then
```